src/drivers/safety_button/SafetyButton.hpp

```diff
@@ -71,15 +73,17 @@ class SafetyButton : public ModuleBase<SafetyButton>, public px4::ScheduledWorkI
 
 private:
 
-	void CheckButton();
+	bool CheckButton();
 	void FlashButton();
 
 
 	uORB::Subscription		_armed_sub{ORB_ID(actuator_armed)};
 	uORB::Publication<safety_s>	_to_safety{ORB_ID(safety)};
+	uORB::PublicationQueued<vehicle_command_s>	_to_command{ORB_ID(vehicle_command)};
 
 	uint8_t				_button_counter{0};
 	uint8_t				_blink_counter{0};
+	bool				_safety_disabled{false};
```

Comment:
The naming of these variables is getting confusing. What is `_safety_disabled` versus `_safetey_off`? This needs to be clear from the names and if not possible from a comment.